app/assets/javascripts/util/url.js.coffee

```diff
@@ -147,6 +139,18 @@ this.edsc.util.url = do(window
     new ChildCompressor('pg', new CwicGranuleIdListCompressor('cx'))
   ]
 
+#   new ParamFlattener(['science_keywords_h', '0', 'category'], 'fsc', false)
+  keywords = []
+  for index in [0...6]
```

Comment:
Scratch what I said... Although CMR is capable of returning an index number that is larger than 6, EDSC limits users to select only one topic which will set the index to always be 0.